lib/modules/manager/tflint-plugin/types.ts

```diff
@@ -0,0 +1,12 @@
+import type { PackageDependency } from '../types';
+
+export interface ExtractionResult {
+  lineNumber: number;
+  dependencies: PackageDependency<TFLintManagerData>[];
+}
+
+export interface TFLintManagerData {
+  pluginName?: string;
+  source?: string;
+  sourceLine?: number;
+}
```

Comment:
i don't see any usage of those manager data

Reply:
I removed it in a previous pull request but I forgot the interface here, I removed it, thanks for pointing out!